src/core/internal/traits.d

```diff
@@ -274,6 +274,23 @@ template hasElaborateDestructor(S)
     }
 }
 
+// std.traits.hasElaboratePostblit
+template hasElaboratePostblit(S)
```

Comment:
What's 'elaborate' about this? Looks like `hasPostblit` to me.

Reply:
You're right. I mistakenly believed `hasPostblit` only looks for a postblit if the type is a struct and is false when the type is an array.